Automattic/wp-calypso
# Post Type List: migrate CSS to webpack

Migrates CSS of the posts list. Very straightforward, no difficulties.

**How to test:**
See the `/posts/site.blog` page for a site. Verify that nothing is broken CSS-wise.

Some parts of the UI active only when the `posts/post-type-list/bulk-edit` feature flag is active. The flag hides a feature that was never finished, but the code is still with us.
